lib/core/file.nit

```diff
@@ -663,6 +663,13 @@ class Path
 		return res
 	end
 
+	# Is `self` the path to a directory ?
```

Comment:
I thinks it is better to state the case of existence because it answer the question about the robustness of the method.

Moreover, it helps the programmer to understand that `path.exists and path.is_dir` is quite redundant.